data/yam/agama/auto/sle_multipath.jsonnet

```diff
@@ -1,6 +1,7 @@
 {
   "product": {
-    "id": "SLES_16.0"
+    "id": "SLES",
```

Comment:
now we use `{{AGAMA_PRODUCT_ID}}`

Reply:
Please rebase, it is now {{AGAMA_PRODUCT_ID}}.

Reply:
this file was done by PR #20894.

Reply:
Yes, so you don't need update it any more after rebase.